full_install.sh

```diff
@@ -1,10 +1,19 @@
 #!/usr/bin/env bash
 set -euo pipefail
-python3 -m pip install -e common/[tests]
-python3 -m pip install -e core/[all,tests]
-python3 -m pip install -e features/
-python3 -m pip install -e tabular/[all,tests]
-python3 -m pip install -e multimodal/[tests]
-python3 -m pip install -e timeseries/[all,tests]
-python3 -m pip install -e eda/
-python3 -m pip install -e autogluon/
+
+# Check if uv is installed
+if ! command -v uv &> /dev/null
```

Comment:
Would there be an issue if we pre-install `uv` in this script?

Reply:
We have to pre-install `uv` if it is not available in the system, otherwise pip install won't work